independent-publisher-connectors/Zoom Meetings/apiDefinition.swagger.json

```diff
@@ -0,0 +1,430 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Zoom Meetings",
+    "description": "This Connector will help to automate Zoom Meeting Operations",
+    "version": "1.0"

```

Comment:
@A-Deekshith The metadata is missing in your connector. Please add it as it is a requirement in the program.
Docs - [Adding Metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-3-add-metadata)